### ISSUE TITLE
sort function for the plugin browser

### ISSUE THREAD

**Issue by KuVa187** [Vencord/plugin-requests#1060]
### Plugin Description

not a plugin but generally a *sort function for the plugin browser*. It would make it much easier to pick the best plugins / plugins you dont have yet if you could sort them by lets say most downloaded or newest for example

### Request Agreement

- [X] I read the above and made sure my plugin is possible and allowed
- [X] I checked the [existing plugins](https://vencord.dev/plugins) and made sure my plugin doesn't already exist
- [X] I [searched the existing issues](https://github.com/Vencord/plugin-requests/issues?q=is%3Aissue) and made sure a similar request doesn't already exist

**Comment by Scyye** [comment]
![image](https://github.com/user-attachments/assets/78f029fc-3b1f-490a-919f-f6dd8b041b03)
like that?

also how would you get the download count of a plugin?

**Comment by mynameisbob1928** [comment]
> also how would you get the download count of a plugin?
The Vencord devs would need to keep a count of probably who has it enabled cuz all plugins are downloaded at install instead of downloads and use some server for an api

**Comment by KuVa187** [comment]
> ![image](https://private-user-images.githubusercontent.com/97131358/359643725-78f029fc-3b1f-490a-919f-f6dd8b041b03.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQxODc0NzEsIm5iZiI6MTcyNDE4NzE3MSwicGF0aCI6Ii85NzEzMTM1OC8zNTk2NDM3MjUtNzhmMDI5ZmMtM2IxZi00OTBhLTkxOWYtZjZkZDhiMDQxYjAzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MjAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODIwVDIwNTI1MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU0ZjU0NTc5NGFiMGRhNzI2YmE0ZmYzZjBjYWZjZjQ1MWExYWZkZDRiODUwYWNiYTllMzI0MGQ3MWJhMWRjY2EmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.xjNPlvCfbrOxT_iKEOnx15Hh-n1C4ptA0-mzqq4ZrOg) like that?
> 
> also how would you get the download count of a plugin?

yeah like that but especially with sort by most downloaded and yes this would mean they have to implement a function to count every download / install from now on. better late than never right?

**Comment by Scyye** [comment]
> better late than never right?

i dont really see why?
doesnt seem very useful outside this (niche) use case

thats just my opinion though.

**Comment by KuVa187** [comment]
well i mean every new user kinda has to pick what plugins to install and theres already like a hundred of them so why not make it much easier for them to just by default sort by most downloaded and give them an easier choice

**Comment by mynameisbob1928** [comment]
Why does it matter what other people have though? Wouldn't it just be better to pick what you prefer?

**Comment by KuVa187** [comment]
this is some seriously cramped "be different" shit. if you dont see how its useful to have the option to sort by most downloaded i can't help you. every single vencord user has to go through the process of installing plugins and USUALLY the plugins downloaded the most are the simply the best or most useful

**Comment by Scyye** [comment]
I- well thats needlessly aggressive, dont ya think?

**Comment by mynameisbob1928** [comment]
> if you dont see how its useful to have the option to sort by most downloaded

It can be useful however it's just not that useful.

> every single vencord user has to go through the process of installing plugins

Pretty much only once though.

> i can't help you

I don't need help :3


If you are really that desperate to know what others think are the best there are some youtube videos that do this as well as their comments.

**Comment by KuVa187** [comment]
nah at this point that just crazy. now i know why no one wants to deal with people like you. if you dont want to implement a function that is present in pretty much every application for a very good reason then don't. i can't be asked to deal with someone like you

**Comment by mynameisbob1928** [comment]
> now i know why no one wants to deal with people like you.

Talking like you know me is not going to do anything

> if you dont want to implement a function

I am not the one implementing the functions, I am merely presenting my opinions and saying what I think in response to what you are saying.

> i can't be asked to deal with someone like you

Yet you decided to write that message and respond

**Comment by Scyye** [comment]
hey, can sm1 lock this, or mark these comments as spam/abuse or smth

**Comment by mynameisbob1928** [comment]
Ok lol I won’t talk unless it is for the plugin itself
